app/scripts/controllers/meeting.details.ctrl.js

```diff
@@ -168,7 +168,7 @@ angular.module('dashboard')
         self.attachmentClicked = function (attachment) {
             setLowerBlockMode(CONST.LOWERBLOCKMODE.ATTACHMENTS);
             if (isMobile) {
-                StorageSrv.setKey(CONST.KEY.SELECTION_DATA, self.aData);
+                StorageSrv.setKey(CONST.KEY.SELECTION_DATA, [self.aData]);
```

Comment:
list.html expects an array (hence the added comment in list ctrl). On signing side it's an array of ListData objects, on meetings side an array with single ListData (unless we combine the buttons on mobile, in which case it would be an array of several ListDatas)